test/e2e/e2e-test/nginx/pom.xml

```diff
@@ -72,7 +72,7 @@
                                             http://${docker.host.address}:${nginx.port}
                                         </url>
                                     </http>
-                                    <time>30000</time>
+                                    <time>130000</time>
```

Comment:
Why need so long time?

Reply:
> Why need so long time?

in my env....

Reply:
Please don't change the upstream timeout, 130k seems too long.

Reply:
OK。